lib/config.js

```diff
@@ -159,6 +158,28 @@ Config.prototype.buildArgs = function () {
     delete args.enable_hangout_services_extension
   }
 
+  if (this.targetOS === 'ios') {
+    args.target_os = 'ios'
+    args.enable_dsyms = false
+    args.enable_stripping = args.enable_dsyms
+    args.use_xcode_clang = args.is_official_build
+    args.use_clang_coverage = false
+    args.is_component_build = false
+    // args.ios_deployment_target '12.0'
```

Comment:
This is required by the iOS lib as we use iOS 12 APIs and our minimum for brave-ios is 12. Is it possible to add this arg just for `brave-core/brave-ios` or does it have to be build-wide?